bundles/binding/org.openhab.binding.velux/README.md

```diff
@@ -94,27 +84,16 @@ As the bridge does not support a real rollershutter interaction, this binding pr
 ### Items velux.items
```

Comment:
This line should say only "Items".  "velux.items" belongs above the code fence that starts the set of examples.